README.md

```diff
@@ -0,0 +1,94 @@
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+
+# Razor
```

Comment:
Can we clean up this file?

Reply:
Readme.md cleaned

related commit - df2a47f66abcd4cec7b23e6bdd96c4fb5de83594